package.json

```diff
@@ -16,7 +16,9 @@
     "dev": "yarn build -- --watch src",
     "build": "babel src --out-dir dst --ignore '**/*.test.js'",
     "lint": "eslint \"{src,example}\"",
-    "test": "yarn lint && jest",
+    "test": "yarn lint && yarn test:unit${TRAVIS:+':coveralls'}",
```

Comment:
nit: Echoing this script gives me `yarn lint && yarn test:unit':coveralls'`. Does it work without quotes?

Reply:
Yeah - actually the single quotes are a typo, and it should be either unquoted or double-quotes.  I think maybe double quotes makes the whole thing a bit more readable, so I'll change it to that :)

Reply:
Oh... but then I've got to escape them which makes the whole thing look messy.  OK - unquoted it is :)